Tests/ExamplesTest/requirements.txt

```diff
@@ -1,17 +1,8 @@
-astroid==2.4.2
-attrs==19.3.0
-isort==4.3.21
-lazy-object-proxy==1.4.3
-mccabe==0.6.1
-more-itertools==8.4.0
-packaging==20.4
-pluggy==0.13.1
-py==1.10.0
-pylint==2.5.3
-pyparsing==2.4.7
-pytest==5.4.3
+attrs==21.4.0
+iniconfig==1.1.1
+packaging==21.3
+py==1.11.0
+pyparsing==3.0.7
+pytest==6.2.5
 scapy==2.4.3
-six==1.15.0
-toml==0.10.1
-wcwidth==0.2.5
-wrapt==1.12.1
```

Comment:
I think most of the "extra" packages are needed for `black` and `pylint` which we might want to add as [pre-commit hooks](https://github.com/seladb/PcapPlusPlus/pull/807) and check them in the CI as well

Reply:
Maybe but the black or pylint commit should introduce them properly.

Reply:
ok, I agree 👍